schemas/stsci.edu/asdf/wcs/icrs_coord-1.0.0.yaml

```diff
@@ -0,0 +1,40 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/wcs/icrs_coord-1.0.0"
```

Comment:
I think we discussed this with regards to `Quantity`, and yes the documentation proposes that tag versions should move in lock-step with the overall ASDF version. However, it seems odd to label something brand new as anything other than 1.0.0, so maybe we can discuss this when we meet.